frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -244,7 +244,11 @@ class AccountsSummary extends Component<Props, State> {
                                         { accounts.length > 0 ? (
                                             accounts.map(account => this.balanceRow(account))
                                         ) : (
-                                            <p>{t('accountSummary.noAccount')}</p>
```

Comment:
why? if accounts.length is 0 it means there are no accounts, not that they are loading.

if you want to distinguish between loading and no accounts, you need to make the accounts undefined before it is set (i.e. treat undefined abd `[]` differently)

Also if if is empty, then won't it route back to `/` again?

https://github.com/digitalbitbox/bitbox-wallet-app/blob/d45cba211340be929f1fbecaaaadebac8ff510ba/frontends/web/src/app.tsx#L139-L146